demo/AppMode.ts

```diff
@@ -71,16 +78,16 @@ function getBaseTableProps(mock: IDataMock) {
         },
         page_action: TableAction.None,
         style_table: {
-            max_height: '800px',
+            maxHeight: '800px',
             height: '800px',
-            max_width: '1000px',
+            maxWidth: '1000px',
             width: '1000px'
         },
         style_cell_conditional: [
-            { max_width: 150, min_width: 150, width: 150 },
-            { if: { column_id: 'rows' }, max_width: 60, min_width: 60, width: 60 },
-            { if: { column_id: 'bbb' }, max_width: 200, min_width: 200, width: 200 },
-            { if: { column_id: 'bbb-readonly' }, max_width: 200, min_width: 200, width: 200 }
+            { if: { }, maxWidth: 150, minWidth: 150, width: 150 },
```

Comment:
that looks weird, but I guess it's valid... is there a reason to put it here rather than `style_cell`?

Reply:
nope, moving it